CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 * Fix incorrect escaping of notebook names ([#566](https://github.com/databrickslabs/terraform-provider-databricks/pull/566))
 * Added support for spot instances on Azure ([#571](https://github.com/databrickslabs/terraform-provider-databricks/pull/571))
 
+**Behavior changes**
```

Comment:
This is not a behavior change, just field that is exposed now

Reply:
changed it to just a line entry